src/main/java/org/broadinstitute/hellbender/tools/ApplyBQSRArgumentCollection.java

```diff
@@ -0,0 +1,79 @@
+package org.broadinstitute.hellbender.tools;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMFileWriter;
+import htsjdk.samtools.SAMFileWriterFactory;
+import htsjdk.samtools.SAMRecord;
+import htsjdk.samtools.util.CloserUtil;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.ArgumentCollection;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.ReadProgramGroup;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.ReadWalker;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.transformers.BQSRReadTransformer;
+import org.broadinstitute.hellbender.transformers.ReadTransformer;
+import org.broadinstitute.hellbender.utils.QualityUtils;
+import org.broadinstitute.hellbender.utils.read.ReadUtils;
+
+import java.io.File;
+import java.io.Serializable;
+import java.lang.annotation.Annotation;
+
+public class ApplyBQSRArgumentCollection implements ArgumentCollection, Serializable {
```

Comment:
This should implement `ArgumentCollectionDefinition` which is the marker interface to label it as an argument collection.  The parser looks for the annotations on the variable it's injecting into, not the type of the variable, so I don't think the doc string will propagate properly.  This is confusing though, so maybe it's worth looking into further.